source/localizable/index.html.erb

```diff
@@ -167,9 +205,9 @@
 
       <div class="form mt-10">
 
-        <form name="contact" class="w-full max-w-lg m-auto" netlify>
-          <div class="flex flex-wrap -mx-3 mb-2">
-            <div class="w-full md:w-1/2 px-3 mb-3 md:mb-0">
+        <form class="w-full max-w-lg m-auto">
```

Comment:
please keep the `name="contact" ` and `netlify` otherwise form submissions don't work